node/plugin/config.go

```diff
@@ -14,6 +14,13 @@ import (
 var (
 	/* Required Flags */
 
+	PubIPProviderFlag = cli.StringFlag{
+		Name:     "public-ip-provider",
+		Usage:    "The ip provider service used to obtain a operator's public IP [seeip (default), ipify)",
+		Required: true,
```

Comment:
Does this need to be required? @shrimalmadhur

Reply:
oh umm - I don't think we should put it as required - can we make this as optional and by default point to `seeip`? 

We can re-use the `.env` one in the setup